src/screens/DiscoveryScreenV2/index.js

```diff
@@ -378,7 +374,9 @@ DiscoveryScreenV2.propTypes = {
 export default DiscoveryScreenV2;
 
 const DiscoveryContainer = ({children}) => {
-  if (Platform.OS === 'ios') return <>{children}</>;
-
-  return <SafeAreaView style={{flex: 1}}>{children}</SafeAreaView>;
+  return (
+    <SafeAreaView style={{flex: 1, backgroundColor: 'white'}}>
+      <KeyboardWrapper>{children}</KeyboardWrapper>
+    </SafeAreaView>
+  );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made to the `DiscoveryContainer` component seem to be aimed at improving the UI design and handling keyboard interactions more effectively. The addition of `backgroundColor: 'white'` will set a white background color for the container, and wrapping children with `KeyboardWrapper` should help manage keyboard display and dismissal. However, the previous version had a condition for iOS platforms. If this condition is still relevant, it should be considered in the new implementation.
```diff
-  if (Platform.OS === 'ios') return <>{children}</>;
-
-  return <SafeAreaView style={{flex: 1}}>{children}</SafeAreaView>;
+  return (
+    <SafeAreaView style={{flex: 1, backgroundColor: 'white'}}>
+      <KeyboardWrapper>{children}</KeyboardWrapper>
+    </SafeAreaView>
+  );
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->